runelite-client/src/main/java/net/runelite/client/util/SwingUtil.java

```diff
@@ -193,7 +199,7 @@ public static boolean isInScreenBounds(final int x, final int y)
 	 * @param callback        the callback
 	 * @param confirmRequired the confirm required
 	 */
-	public static void addGracefulExitCallback(@Nonnull final JFrame frame, @Nonnull final Runnable callback, @Nonnull final Callable<Boolean> confirmRequired)
+	public static void addGracefulExitCallback(final RuneLiteConfig config, @Nonnull final JFrame frame, @Nonnull final Runnable callback, @Nonnull final Callable<Boolean> confirmRequired)
```

Comment:
Might as well tag `@Nonnull` onto `config` so it matches the rest of the method signature.